src/collections/_documentation/error-reporting/getting-started-verify/browser.md

```diff
@@ -1,4 +1,6 @@
-One way to break your JavaScript application is to call an undefined function:
+What better way to send your first event than to break your application!
+
+One way to do it is to call an undefined function:
```

Comment:
```suggestion
Calling an undefined function will trigger a break:```

Reply:
"trigger a break" sounds really strange. "break" in JS is a keyword unrelated to exceptions.

How about "throw an exception", instead?

```suggestion
Calling an undefined function will throw an exception:
```

Reply:
"throw an exception" is great!